contrib/golang/filters/http/source/golang_filter.cc

```diff
@@ -717,7 +717,8 @@ uint64_t FilterConfig::getConfigId() {
   ASSERT(dlib != nullptr, "load at the config parse phase, so it should not be null");
 
   std::string str;
-  ASSERT(plugin_config_.SerializeToString(&str));
+  auto res = plugin_config_.SerializeToString(&str);
```

Comment:
is this from a merge ?

if not i think separate out these changes to a different PR

Reply:
yes, It's cherry pick from PR https://github.com/envoyproxy/envoy/pull/25227

Reply:
ah i see - could you merge main then please

Reply:
Ok. I'll do it.